packages/wxt/src/core/builders/vite/plugins/extensionApiMock.ts

```diff
@@ -32,11 +32,21 @@ export function extensionApiMock(config: ResolvedConfig): vite.PluginOption {
         },
       };
     },
-    resolveId(id) {
-      if (id.endsWith(virtualSetupModule)) return resolvedVirtualSetupModule;
+    resolveId: {
+      filter: {
+        id: new RegExp(`${virtualSetupModule}$`),
+      },
+      handler() {
+        return resolvedVirtualSetupModule;
+      },
     },
-    load(id) {
-      if (id === resolvedVirtualSetupModule) return setupTemplate;
+    load: {
+      filter: {
+        id: new RegExp(`^${resolvedVirtualSetupModule}$`),
```

Comment:
```suggestion
        id: resolvedVirtualSetupModule,
```

Is this the same or different? Does rolldown do a partial match for regular strings or a full match?

Reply:
TLDR;

We won't able to use strings as i've though in:
https://github.com/wxt-dev/wxt/pull/1787#issuecomment-3190878288

It's because we want to check direct string in imports, not resolved absolute path from bundling process.

We need to use RegExp everywhere as it's now.

I was trying to prettify it to make it with /expression/, but it wasn't working with vars, that's why i need to use `new RegExp()` syntax :)